### PR TITLE
add zero-grad for rounding ops

### PR DESCRIPTION
**Pull request by szha**
## Description ##
add zero gradients to rounding operators including 'rint', 'ceil', 'floor', 'trunc', and 'fix'.

### Essentials ###
- [x] Passed code style checking (`make lint`)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [x] register FGradient with MakeZeroGradNodes for 'rint', 'ceil', 'floor', 'trunc', 'fix'

### CODE REVIEW

**Comment by piiswrong** [comment]
Are these zero for other frameworks?

**Comment by szha** [comment]
TF has None which means zero https://github.com/tensorflow/tensorflow/issues/5888

**Comment by szha** [comment]
pytorch has zeros for round
```python
In [1]: import torch

In [2]: from torch.autograd import Variable

In [3]: x = Variable(torch.randn(4), requires_grad=True)

In [4]: y = x.round()

In [5]: z = y.sum()

In [6]: z.backward()

In [7]: x.grad
Out[7]:
Variable containing:
 0
 0
 0
 0
[torch.FloatTensor of size 4]
```
